spec/features/all_users_can_submit_applications_to_become_store_admins_spec.rb

```diff
@@ -20,7 +20,7 @@
   fill_in 'First Name', with: "Ziba"
   fill_in 'Last Name', with: "Andersen"
   fill_in 'Email', with: "#{rand(999)}@puppy.com"
-  fill_in 'Phone Number', with: "1-800-woo ooof"
+  fill_in 'Phone Number', with: "1234567890"
```

Comment:
I like how you looked back into your tests to accommodate the new validations set.